src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PageType.php

```diff
@@ -27,37 +27,73 @@
 namespace PrestaShopBundle\Form\Admin\Configure\ShopParameters\ProductPreferences;
 
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
-use Symfony\Component\Form\AbstractType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\GreaterThanOrEqual;
+use Symfony\Component\Validator\Constraints\Type;
 
 /**
  * Class generates "Product page" form
  * in "Configure > Shop Parameters > Product Settings" page.
  */
-class PageType extends AbstractType
+class PageType extends TranslatorAwareType
 {
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('display_quantities', SwitchType::class)
-            ->add('display_last_quantities', IntegerType::class)
-            ->add('display_unavailable_attributes', SwitchType::class)
-            ->add('allow_add_variant_to_cart_from_listing', SwitchType::class)
+            ->add('display_quantities', SwitchType::class, [
+                'label' => $this->trans('Display available quantities on the product page', 'Admin.Shopparameters.Feature'),
+            ])
+            ->add('display_last_quantities', IntegerType::class, [
+                'label' => $this->trans('Display remaining quantities when the quantity is lower than', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Set to "0" to disable this feature.', 'Admin.Shopparameters.Help'),
+                'constraints' => [
+                    new Type(
+                        [
+                            'value' => 'numeric',
+                            'message' => $this->trans('The field is invalid. Please enter a positive integer number.', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                    new GreaterThanOrEqual(
+                        [
+                            'value' => 0,
+                            'message' => $this->trans('The field is invalid. Please enter a positive integer number.', 'Admin.Notifications.Error'),
+                        ]
+                    ),
+                ],
+            ])
+            ->add('display_unavailable_attributes', SwitchType::class, [
+                'label' => $this->trans('Display unavailable attributes on the product page', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('If an attribute is not available in every product combination, it will not be displayed.', 'Admin.Shopparameters.Help'),
```

Comment:
Is this wording new?

Reply:
Here too, shouldn't the default status be "No"? Because in the help text we say that if an attribute is not available it will not be displayed but at the same time the button is automatically set on "Yes"